packages/babel-parser/src/plugins/flow.js

```diff
@@ -422,8 +421,7 @@ export default (superClass: Class<Parser>): Class<Parser> =>
     flowParseDeclareModuleExports(
       node: N.FlowDeclareModuleExports,
     ): N.FlowDeclareModuleExports {
-      this.expectContextual("module");
-      this.expect(tt.dot);
```

Comment:
I might be wrong, but this was eating two tokens? Now it only skip one.

Reply:
right the other is skipped outside.